eva_cttv_pipeline/evidence_string_generation/evidence_strings.py

```diff
@@ -205,7 +184,16 @@ def __init__(self, clinvar_record, clinvar_record_measure, report, trait, conseq
             self.unique_reference = ref_list[0]
 
         if clinvar_record.clinical_significance:
-            self.clinical_significance = clinvar_record.clinical_significance
+            self.clinical_significance = process_clinical_significance(clinvar_record.clinical_significance)
+
+        # Populate star rating and review status
+        star_rating, review_status = clinvar_record.score
+        self.clinvar_rating = (star_rating, review_status)
+
+        # Populate mode of inheritance (if present)
+        mode_of_inheritance = clinvar_record.mode_of_inheritance
+        if mode_of_inheritance:
+            self.mode_of_inheritance = mode_of_inheritance
```

Comment:
Not hugely important but I would move the if statement into the setter as I would argue the treatment of empty mode of inheritance should be managed there (unless it needs to be nullable)
```suggestion
        self.mode_of_inheritance = clinvar_record.mode_of_inheritance
```

Reply:
Thanks! It does sound like a job for a setter. I've applied the suggestions